### ISSUE TITLE
Input with rounded edges is not showing correct when I add <b-form-invalid-feedback>

### ISSUE THREAD

**Issue by leo95works** [bootstrap-vue/bootstrap-vue#5314]
I have a small question and I don't know if it is a bug. I have the following form:
```
....
       <b-form>
            <b-card no-body>
              <b-card-header />
              <b-card-body>
                <validation-provider v-slot="errors" name="email"> # Vee-Validate
                  <b-form-group>
                    <b-input-group>
                      <b-input-group-prepend>
                        <b-input-group-text>
                          <fa-icon icon="user" /> # Font-Awesome Icon
                        </b-input-group-text>
                      </b-input-group-prepend>
                      <b-form-input
                        id="email"
                        v-model="form.email"
                        type="text"
                        :state="check(errors)"
                        :placeholder="$t('general.email')"
                        :autofocus="true"
                      />
                      <b-form-invalid-feedback>
                        {{ errors.errors[0] }}
                      </b-form-invalid-feedback>
                    </b-input-group>
                  </b-form-group>
......
```
![image](https://user-images.githubusercontent.com/60673955/81232877-af42b280-8fc3-11ea-80fb-18a75ab9f907.png)

Notice in the input how the corners of the edges on the right are not rounded:

![image](https://user-images.githubusercontent.com/60673955/81233026-fcbf1f80-8fc3-11ea-91f7-fd2c902a7578.png)

but if I remove the component ``` <b-form-invalid-feedback> ``` the edges are as they should be, rounded:

![image](https://user-images.githubusercontent.com/60673955/81233149-28420a00-8fc4-11ea-87fa-6ff103446213.png)

![image](https://user-images.githubusercontent.com/60673955/81233163-2c6e2780-8fc4-11ea-9db9-cbadfec53246.png)

my question is, is this normal or is it a bug? I have defined true the variable ```$enable-rounded``` in my scss file. 

Thank you very much in advance

**Comment by tmorehouse** [comment]
Bootstrap's Input group CSS doesn't support the invalid/valid feedback classes inside them.  Place the feedback after the input group, and set the `state` prop on the feedback component to be the same `state` you apply to the form-input.

**Comment by tmorehouse** [comment]
The CSS uses `:last-child` to set the corner rounding (the rounding is not done on the input-group itself, but on the elements within it).  Since your feedback is now the last element and the input is no longer the last element, the input has it's rounding disabled.

**Comment by leo95works** [comment]
@tmorehouse  Thanks, following your advice, now it looks as expected. Thank you.

**Comment by mlpinit** [comment]
TL;DR You can also use `has-validation` next to `input-group`

ℹ️ Stumbled upon this while looking into the same issue. For whomever this may help, notice that when using the invalid-feedback class, you can ensure corners are rounded by applying the `has-validation` class to the `input-group`:

e.g: 

```
   <div class="input-group has-validation">
      ...
      <div class="invalid-feedback">
         "Some validation message here"
      </div>
  </div>
```

Even better examples can be found in the documentation: https://getbootstrap.com/docs/5.0/forms/validation/
